collagraph/component.py

```diff
@@ -0,0 +1,41 @@
+from abc import ABCMeta, abstractmethod
+
+from observ import reactive, readonly
+
+
+class Component(metaclass=ABCMeta):
+    """Abstract base class for components"""
+
+    def __init__(self, props=None):
+        self.props = readonly(props or {})
+        self.state = reactive({})
+
+    def mounted(self):
+        """Called after the component has been mounted.
+
+        A component is considered mounted after:
```

Comment:
You might want to be very clear here that this means the hook `is called after:`

Reply:
The doc string already states this (line 14), so I'm not sure what you're getting at. Does that suffice?